submit_cgap/tests/test_submission.py

```diff
@@ -1214,37 +1216,69 @@ def get_time_after_wait(self):
     def make_uploaded_lines(self):
         uploaded_time = self.get_time_after_wait()
         return [
-            f"The server {SOME_SERVER} recognizes you as J Doe <jdoe@cgap.hms.harvard.edu>.",
+            f"The server {SOME_SERVER} recognizes you as: J Doe <jdoe@cgap.hms.harvard.edu>",
             (
-                f"{uploaded_time} Bundle uploaded, assigned uuid {SOME_UUID} for tracking."
-                " Awaiting processing..."
+                (f"{uploaded_time} Bundle uploaded."
+                 f" Checking ingestion process using IngestionSubmission uuid: {SOME_UUID} ...")
             ),
         ]
 
-    def make_wait_lines(self, wait_attempts):
+    def make_wait_lines(self, wait_attempts, outcome: str = None):
         result = []
         uploaded_time = self.get_time_after_wait()
-        for idx in range(wait_attempts):
-            time_delta_from_start = (PROGRESS_CHECK_INTERVAL + self.wait_time_delta) * (idx + 1)
+        time_delta_from_start = 0
+        nchecks = 0
+        CLEAR = "\033[K"
```

Comment:
Maybe call this variable `ERASE_TO_EOL`? I panicked that you were going to erase my whole window output and had to go look up the effect. Perhaps also a comment:

```suggestion
        ERASE_TO_EOL = "\033[K"  # Ref: https://tldp.org/HOWTO/Bash-Prompt-HOWTO/x361.html
```
If length is sn issue, I'd bargain down to:
```
        CLEAR = "\033[K"  # Erase to EOL. Ref: https://tldp.org/HOWTO/Bash-Prompt-HOWTO/x361.html
```
If the line is too long, add `  # noQA` (python comment syntax, not query URL syntax, note leading space) to the end of it so you don't have to break the URL